server/server.go

```diff
@@ -251,24 +250,18 @@ func handleWebsocket(w http.ResponseWriter, r *http.Request) {
 	c.Listen()
 }
 
-func getUserIdFromCookie(r *http.Request) UserId {
-	cookie := r.Header.Get("Cookie")
-	parsedCookie, cookieErr := url.ParseQuery(cookie)
-	if cookieErr != nil {
-		log.Println("Cookie", cookieErr)
+func mintCookie(userId UserId) *http.Cookie {
+	c := &http.Cookie{
+		Name:     cookiename,
+		Value:    string(userId),
+		MaxAge:   86400,
+		SameSite: http.SameSiteLaxMode,
```

Comment:
This makes way more sense than the mysterious `2` I had in there before. Thanks for making the code so much clearer!